### DISCUSSION TITLE
VTOL Wingspan

### DISCUSSION THREAD

**Discussion by socalengineer** [StephenCarlson/MiniHawk-VTOL#40] [General]
Hi Stephen,

Why did you choose the wingspan you did for the MiniHawk?

If you had to choose the optimal wingspan for:
1. Flight time of 2hrs and be able to carry a 12000mah battery or very large battery for flight time. I was thinking 4s2p or 4s3p made with these cells:
https://www.18650batterystore.com/products/molicel-p42a
2. Good flight handling characteristics for FPV plane (include some electronics raspberry pi zero - 30g, cube orange flight controller-73g, GoPro Hero 7 Naked - 26g)
3. Be able to carry solar panels
4. Be able to 3d print
5. Smallest possible wingspan meeting the above requirements
6. Able to fly well in winds 15mph or 30mph

What would you choose?

I was thinking 1.2m - 1.3m

**Comment by StephenCarlson** [comment]
Hi @socalengineer !

First, the query on why the current dimensions: The MiniHawk was designed as the _minimum_ possible demonstrator for a 3D-printed solar-powered migratory VTOL. I suppose it is safe for me to post the design paper I wrote for it a year ago: 
[ICUAS21_0155_FI.pdf](https://github.com/StephenCarlson/MiniHawk-VTOL/files/8112215/ICUAS21_0155_FI.pdf) . Basically, it is sized to accommodate most of the standard 5-inch-class quadrotor motors on the market today, and offer sufficient payload capacity and avionics bay volume for experiments, but I am the first to admit, it really is a brick with wings; it is not efficient and only flies for a few minutes. You are not the first to suggest a larger one; I've been hounded by my advisor to consider making a larger more capable one as well.

So, lets see, 2 hours flight time, 12 Ah battery using 18650's or whatever the current fade is, and payload for a stripped-down GoPro and such. I'm going to make a guess that the avionics and battery will be at least 1kg. (The 4s2p will be 560g.) If the airframe takes 1/3 of the final total weight, and the motors and servos occupy another 1/3, that will be at least 3 kg. Our wing area and aspect ratio will determine how much it does not fly like a brick, and the solar cells will interact with whatever wing shape we select. I've already decided that a nice round number of solar cells will be either 24 or 36 of the Maxeon C60 form-factor, in 2 or 3 rows. That will be either 12 or 18 cells of length on the top wing, plus center body and winglets. Lets stick to 18 cells of length, since we want a higher aspect ratio wing. Also, since solar cells will be on the wing surface, we cannot get to aggressive with the wing taper or washout; basically, we have to have a rectangular wing to avoid warping the solar cells too much. So, 18 cells * 125mm each, plus about 160mm for the center body and 100mm for the winglets * 2. This is about 2600mm wingspan. If we revert to just 12 solar cells of length with either 2 rows (24 cells) or 3 rows (36 cells), that gets us a wing that will be about 1800mm. We might have to abandon the plank planform that the MiniHawk currently uses and either do a swept delta wing, or a conventional twin-boom or just straight conventional single tail boom.

Ok, this is still too big. To get down to 1200mm, I think we can maintain 1 row of solar cells, with 6 instead of the current 4, still in 1 row. We would have a higher aspect ratio, so better efficiency, and if we abandon the plank planform and go with a swept or conventional aircraft, we can use better airfoils to try to get the 2 hours flight time on 12 Ah. Note that cruise has to be 6 Amps to make it 2 hours on 12 Ah, the current design flies between 8 and 12 Amps on a 4s. But as to the wind penetration, 30 mph is 13.4 m/s, and that is about the airspeed we would need to fly at for best endurance, so this is cutting it a bit close.

This is fun, lets continue batting this around. How much do we want to keep the current planform/shape? Is my estimate for the final aircraft weight fair, or is 2kg or less likely?

**Comment by socalengineer** [comment]
Hi Stephen, yes I read through your paper. Excellent work.

The plan was to go for ideal weight for flight characteristics. The 2hrs is wishful thinking for the flight time we will have to make a compromise somewhere so that it has decent flight abilities. Compromises would be 4s2p instead of 4s3p so that we could accommodate the weight of the solar cells. I'm assuming < 1.8kg is ideal for this setup.

I have seen a lot of YouTube videos of different platforms, and have found that the Y3 with a conventional single tail boom is the most simple setup to get the project off the ground, and most efficient. It would also use a V-tail design in the rear. Here are some aircraft with 1.2m-1.3 wingspan with the configuration I am talking about. Basically I want to create something very similar to these but with solar power on landing, and for FPV purposes:

https://www.youtube.com/watch?v=ZePxfRYrRCg&t=1s
https://fae-drones.com/FAE-VTOL-COLIBRI-DRONE-76.html

https://www.youtube.com/watch?v=SD7mFplgriU
https://www.flightwave.aero/edge-130-uas/

https://www.youtube.com/watch?v=oz_T7Kt4YX8


This is also popular, but 1.8m a bit big for hobby purposes:
https://www.youtube.com/watch?v=db3pM74u6mM
https://www.youtube.com/watch?v=arQNWQrE06w

**Reply by StephenCarlson** [reply]
Toward the battery solution, is this equivalent to the cell you posted? https://www.racedayquads.com/collections/4s-batteries/products/upgrade-energy-dark-lithium-14-8v-4s2p-21700-8200mah-25c-li-ion-battery-xt60

**Reply by socalengineer** [reply]
Yes it is. Not sure why they listed it as 8200mah? It should be 8400mah by my calculation in 4s2p configuration. Also I calculated ~11.7C or so. Not sure how they got 25C.
